GPflow/priors.py

```diff
@@ -88,6 +104,10 @@ def __init__(self, mu, sigma):
     def logp(self, x):
         return tf.reduce_sum(densities.laplace(self.mu, self.sigma, x))
 
+    def sample(self, shape=()):
+        return np.array(np.random.laplace(self.mu, self.sigma,
+                                          size=shape if shape else None))
```

Comment:
Checked what the default is for scalar Parameters in GPflow, and that's actually (1,) so I'll modify it to fit that standard.